### DISCUSSION TITLE
Time Look Up Table

### DISCUSSION THREAD

**Discussion by kmerbcn** [OpenModelica/OpenModelica#13204] [Q&A] [answered]
Hello to all,

I am not sure if this topic was already asked here. I checked in advanced, but I did not find anything matching.
My question is if there is already a block in Open Modelica performing Look Up tables without interpolation.
I have found TimeTable at Blocks -> Sources -> TimeTable, but I do not want any sort of interpolation between elements in time.
I want a pure discontinous function.

Best regards,

Javier

**Comment by casella** [comment, marked as answer]
Unfortunately not, probably because piecewise-constant interpolation was deemed too crude by the implementors of the TimeTable model.

I guess you can program such a function yourself in pure Modelica easily.
